README.md

```diff
@@ -19,3 +19,8 @@ A workflow manager for curating MRI and tabular data and standardized processing
 - [fMRIPrep](https://fmriprep.org/en/1.5.5/singularity.html) 
 - [TractoFlow](https://github.com/scilus/tractoflow)
 - [MAGeT Brain](https://github.com/CoBrALab/MAGeTbrain)
+
+### GH workflow for contributing to template and dataset-specific forks
+
+![mr_proc_gh_contribution_workflow](https://user-images.githubusercontent.com/7978607/226065220-c1123b5e-8774-4f02-b8ef-0f2255cdf122.jpg)
+
```

Comment:
Is the direction of the arrow  mr_proc-[dataset] <-> mr_proc-(*) ok ? are you not pushing the merged feature on mr_proc-[dataset] to mr_proc-(*) ? (at the moment, arrow seems to indicate you will push from  mr_proc-(*) to mr_proc-[dataset] )

Reply:
Yes, nothing should be pushed from `mr_proc-[dataset]` to `mr_proc(*)`. Updates to `mr_proc(*)` will come from user-GH fork PRs only and the distributed to all the datasets by GH-actions. This is to keep "global" changes separate from "dataset-specific" changes.